tests/setup_test.go

```diff
@@ -101,8 +101,12 @@ const (
 )
 
 const (
-	remoteErrorNotExist   = "remote error notexist"
-	remoteErrorBadRequest = "remote error badrequest"
+	remoteErrorNotExist         = "remote error notexist"
+	remoteErrorBadRequest       = "remote error badrequest"
+	remoteErrorForbiddenRequest = "remote error forbidden"
+	remoteErrorUnauthorized     = "remote error unauthorized"
+	remoteErrorUnknown          = "unknown api error (status: 403)"
+	remoteErrorInternal         = "remote error internal"
```

Comment:
I meant only the strings for the remoteErrorForbidden you added. The others should be done in another PR...